README.md

```diff
@@ -2,8 +2,8 @@
 
 This plugin allows a user to subscribe a channel to an RSS (Version 2 only) or an Atom Feed.
 
-Version 0.1.0 requires Mattermost 5.10
-Version < 0.1.0 requires Mattermost 5.6
+- Version 0.1.0+ requires Mattermost 5.10
+- Version < 0.1.0 requires Mattermost 5.6
```

Comment:
@wbernest Can we use the plugin 0.1.0 and 0.2.0 with Mattermost 5.11 or higher?
Can we use the plugin < 0.1.0 with Mattermost 5.7 or higher?

Reply:
I think < 0.1.0 needs to be less than Mattermost 5.7.

Reply:
Thanks. I am adding information as such in another PR.